### DISCUSSION TITLE
Do we have a component built out yet for a calendar?

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#1380] [Q&A] [answered]
I am a new designer over Marketing Campaign and I am in middle of redesigning something and I would love to use Paste component to replace some of the legacy SendGrid stuff but seems I can't find a calendar component in Paste yet. I am wondering if that's on the roadmap for the team and if not what's the best way for me to contribute. Thanks

**Comment by zahnster** [comment, marked as answer]
Hi @frankfzhao,

We are actively working on a "Date Picker" component for our next release, scheduled to go out on June 1st, 2021.

You can find our roadmap plans here: https://paste.twilio.design/roadmap
